### DISCUSSION TITLE
guide-R/bases/attributs

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#31] [Forum]
# guide-R/bases/attributs

https://larmarange.github.io/guide-R/bases/attributs.html

**Comment by rphlmm** [comment]
'length' et non 'lenghth' (paragraphe introductif)

**Reply by larmarange** [reply]
Merci du retour. Ce sera corrigé dans la prochaine version.

**Reply by Djidjoho1298** [reply]
Il n'y a pas possibilité d'avoir le pdf du document ?

**Reply by larmarange** [reply]
L'export PDF a été désactivé il y a quelques mois en raison de problèmes techniques. 

Il faut que je refasse des tests

**Reply by larmarange** [reply]
L'export PDF a été désactivé il y a quelques mois en raison de problèmes techniques. 

Il faut que je refasse des tests

**Comment by Djidjoho1298** [comment]
D'accord, merci pour le retour.

Espoir que nous l'aurons d'ici peu
Merci pour tout

___________________________________
Djidjoho Christian TANDJIEKPON
Epidémiologiste Biostatisticien
Secrétaire Général du Réseau des professionnels en Santé publique et en Épidémiologie du Bénin (RéSEB)
Tél : +229 01 62 98 74 02

Adresse mail : ***@***.******@***.***>



________________________________

**Reply by larmarange** [reply]
Bonjour. L'export PDF est toujours problématique. Mais vous disposez maintenant d'un export au format ePUB permettant de tout télécharger dans un seul fichier : https://larmarange.github.io/guide-R/guide-R.epub

Vous pouvez trouver de nombreux lecteurs epub gratuit sur le net, comme par exemple https://chromewebstore.google.com/detail/epub-reader-pour-google-c/fnplkbhndemgbopkkpmpnfklkhphpneg?hl=fr

**Comment by Djidjoho1298** [comment]
Merci infiniment Joseph Larmarange
Merci pour ce travail formidable que vous accomplissez

___________________________________
Djidjoho Christian TANDJIEKPON
Epidémiologiste Biostatisticien
Secrétaire Général du Réseau des professionnels en Santé publique et en Épidémiologie du Bénin (RéSEB)
Tél : +229 01 62 98 74 02

Adresse mail : ***@***.******@***.***>



________________________________
De : Joseph Larmarange ***@***.***>
Envoyé : vendredi 30 mai 2025 18:08
À : larmarange/guide-R ***@***.***>
Cc : Djidjoho1298 ***@***.***>; Comment ***@***.***>
Objet : Re: [larmarange/guide-R] guide-R/bases/attributs (Discussion #31)


Bonjour. L'export PDF est toujours problématique. Mais vous disposez maintenant d'un export au format ePUB permettant de tout télécharger dans un seul fichier : https://larmarange.github.io/guide-R/guide-R.epub

Vous pouvez trouver de nombreux lecteurs epub gratuit sur le net, comme par exemple https://chromewebstore.google.com/detail/epub-reader-pour-google-c/fnplkbhndemgbopkkpmpnfklkhphpneg?hl=fr
